### ISSUE TITLE
Persistent `background.page` is not allowed, expects `background.scripts` property

### ISSUE THREAD

**Issue by scholtzm** [mozilla/addons-linter#4400]
### Describe the problem and steps to reproduce it:

Have a manifest.json with `background.page` property and `persistent` set to `true`.

### What happened?

Addons linter throws error:

```
MANIFEST_FIELD_REQUIRED   
"/background" must have required property 'scripts'
See https://mzl.la/1ZOhoEN (MDN Docs) for more information.
```

### What did you expect to happen?

According to docs, `page` property should be allowed and work correctly together with persistent flag in Mv2.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background

**Comment by rpl** [comment]
@scholtzm would you mind to attach in a comment the manifest.json file you are validating?

I just tried with the following manifest.json file content:
```
{
  "manifest_version": 2,
  "name": "test-background-persistent-linter-error",
  "version": "0.1",
  "background": {
    "page": "background.html",
    "persistent": true
  }
}
```

and then executed the last addons-linter release (v5.10.0) on it and it didn't report any error, warning or notice, and so it is possible there is something missing in the manifest to be able to trigger the same issue that you are experiencing.

**Comment by scholtzm** [comment]
Misspelled the word `persistent` in the manifest and the error message threw me off the track. Thanks for quick response!
